test/fixtures/xml/userTask-zeebe-taskListener.part.bpmn

```diff
@@ -0,0 +1,11 @@
+<bpmn:task
```

Comment:
Let's make this match the expectations. We want the task listeners to be allowed in `bpmn:userTask` only.

Reply:
BTW how come the test is passing if we read a task here and check for user task as the type?

Reply:
it passes for `bpmn:userTask` and `bpmn:task` but does not for `bpmn:endEvent` for example 🤔